roles/libvirt_manager/tasks/create_vms.yml

```diff
@@ -59,7 +59,7 @@
   block:
     - name: Load OCP VMs
       ansible.builtin.include_tasks: ocp_xml.yml
-      loop: "{{ vm_data.value.xml_paths }}"
+      loop: "{{ vm_data.value.xml_paths | sort }}"
```

Comment:
yes, we have a problem if the paths are from different director. Nor do we have access to VM name. I think for now `sort` is a simple option.

Also, we control the path the xml is being created in case of OCP. This type is notorious

Reply:
Yeah, if you want something quick this will do the job.